docs/index.rst

```diff
@@ -5,29 +5,26 @@ JAX is Autograd_ and XLA_, brought together for high-performance numerical compu
 
 .. grid:: 3
 
-   .. grid-item::
+   .. grid-item-card:: Familiar API
+      :columns: 12 6 6 4
```

Comment:
The meaning of these numbers is described here: https://sphinx-design.readthedocs.io/en/latest/grids.html#introduction

Namely, on extra small screens, each card will occupy the full screen width. On medium and large screens, each card will occupy half the screen width. On extra large screens, each card will occupy 1/3 of the screen width.